assets/samplesheet.csv

```diff
@@ -1,3 +1,3 @@
-sample,fastq_1,fastq_2
-SAMPLE_PAIRED_END,/path/to/fastq/files/AEG588A1_S1_L002_R1_001.fastq.gz,/path/to/fastq/files/AEG588A1_S1_L002_R2_001.fastq.gz
-SAMPLE_SINGLE_END,/path/to/fastq/files/AEG588A4_S4_L003_R1_001.fastq.gz,
+sample,fastq_1,fastq_2,N_barcodes,barcode_file,sample_size,len_barcode,Nthresh,Ntriage
+S10,/user/gent/446/vsc44685/ScratchVO_dir/ST_transcriptomics_local/test/S10_UHRR_R1.fastq.gz,/user/gent/446/vsc44685/ScratchVO_dir/ST_transcriptomics_local/test/S10_UHRR_R2.fastq.gz,34500,/user/gent/446/vsc44685/ScratchVO_dir/panoramaseq/nf-core-panoramaseq/assets/barcodes_coords.csv,40000,36,9,1000
+S09,/user/gent/446/vsc44685/ScratchVO_dir/ST_transcriptomics_local/test/S09_UHRR_R1.fastq.gz,/user/gent/446/vsc44685/ScratchVO_dir/ST_transcriptomics_local/test/S09_UHRR_R2.fastq.gz,34500,/user/gent/446/vsc44685/ScratchVO_dir/panoramaseq/nf-core-panoramaseq/assets/barcodes_coords.csv,40000,36,9,1000
```

Comment:
These local paths should be updated to urls so that this samplesheet works anywhere